tests/Unit/Infrastructure/Twig/AppExtensionTest.php

```diff
@@ -44,71 +44,72 @@ public function testFormatDateTimeWithTime(): void
         );
     }
 
-    private function provideIsFuture(): array
+    private function provideIsPast(): array
     {
         return [
             // Day after
             [
                 'date' => '2023-01-07',
-                'time' => null,
-                'now' => '2023-01-06 10:30:00',
-                'result' => true,
+                'now' => '2023-01-06',
+                'result' => false,
             ],
+            // Day before
             [
-                'date' => '2023-01-07',
-                'time' => '09:00:00',
-                'now' => '2023-01-06 10:30:00',
+                'date' => '2023-01-05',
+                'now' => '2023-01-06',
                 'result' => true,
             ],
+            // Same day
+            [
+                'date' => '2023-01-06',
+                'now' => '2023-01-06',
+                'result' => false,
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider provideIsPast
+     */
+    
+    public function testIsClientPastDay(string $date,string $now, bool $result): void
+    {
+        $date = new \DateTimeImmutable($date);
+        $now = new \DateTimeImmutable($now);
+        $this->assertSame($result, $this->extension->isClientPastDay($date, $now));
+    }
+    
+    private function provideIsFuture(): array
+    {
+        return [
+            // Day after
             [
                 'date' => '2023-01-07',
-                'time' => '00:00:01',
                 'now' => '2023-01-06',
                 'result' => true,
             ],
             // Day before
             [
                 'date' => '2023-01-05',
-                'time' => null,
-                'now' => '2023-01-06 10:30:00',
-                'result' => false,
-            ],
-            [
-                'date' => '2023-01-05',
-                'time' => '23:59:59',
                 'now' => '2023-01-06',
                 'result' => false,
             ],
             // Same day
             [
                 'date' => '2023-01-06',
```

Comment:
idem on peut mettre des heures dans ce cas pour vérifier la réinitialisation de l'heure

```suggestion
            // Same day (time does not matter)
            [
                'date' => '2023-01-06T08:30:00',
```